### DISCUSSION TITLE
Checks still down email - show checks down first or only those checks that are down

### DISCUSSION THREAD

**Discussion by gary-ionline** [healthchecks/healthchecks#878] [Ideas]
Firstly - great product. I really enjoy the simplicity and ease of use of Healthchecks. 

For this email alert though, I have often looked at the "Reminder: X checks still down" email and found it confusing to see what checks are in fact still down. We're up to 60 or so checks at the moment. 

Currently the entire list of checks if given with the latest status, but in the past I would have found it more useful to have only the 2 or 3 checks that are down to see if I need to action or if it's a known issue. Sometimes projects are in progress of being deployed and they are often up and down until fully into production. This can cloud the problem when we have issues on real production issues. 

A lot of the mails are read on a phone when you're least expecting them, so a short summary of all checks down will really help. The long format and list just doesn't come out well. It's also a really useful reminder especially after a massive event overnight (server reboot / internet outage) to get a summary of what's still down. 

Thoughts?

**Comment by cuu508** [comment]
Thanks for the suggestion, that is a good idea. I too have scrolled through the list looking for that one check that is down and triggering the hourly reminder. 

Implemented in https://github.com/healthchecks/healthchecks/commit/646f79c54e86971e7528dbe43e14f5b726048e32

**Reply by gary-ionline** [reply]
Hi @cuu508 
That's great news!
Looking forward to it
Thank you!

**Reply by cuu508** [reply]
This change is deployed to https://healthchecks.io, and will be in the next release of the self-hosted version.
